components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -595,9 +595,7 @@ public DataSection documentInfoHeader(Set<String> enabledFields) throws Exceptio
 
     public DataSection documentInfoBody(Patient patient) throws Exception
     {
-        @SuppressWarnings("deprecation")
-        DocumentAccessBridge dab = Utils.getComponent(DocumentAccessBridge.class);
-        XWikiDocument patientDoc = (XWikiDocument) dab.getDocument(patient.getDocument());
+        XWikiDocument patientDoc = patient.getXDocument();
```

Comment:
Note: it seems like excel export is initiated from velocity, and `patient` in an instance of `SecurePatient` here, which breaks things. Need to fix